railties/test/application/rake/dbs_test.rb

```diff
@@ -68,6 +68,64 @@ def db_create_and_drop(expected_database, environment_loaded: true)
         db_create_and_drop("db/development.sqlite3", environment_loaded: false)
       end
 
+      test "db:create and db:drop respect environment setting with multiline ERB" do
+        app_file "config/database.yml", <<-YAML
+          development:
+            database: <%=
+              Rails.application.config.database
+            %>
+            adapter: sqlite3
+        YAML
+
+        app_file "config/environments/development.rb", <<-RUBY
+          Rails.application.configure do
+            config.database = "db/development.sqlite3"
+          end
+        RUBY
+
+        db_create_and_drop("db/development.sqlite3", environment_loaded: false)
+      end
+
+      test "db:create and db:drop respect environment setting in with YAML containing multiline and split erb" do
+        app_file "config/database.yml", <<-YAML
+          development:
+          <%
+            if Rails.application.config.database
+          %>
+            database: <%= Rails.application.config.database %>
+          <% else %>
+            database: db/default.sqlite3
+          <%
+            end
+          %>
```

Comment:
Can you bring the if and end ERB onto one line?

`<% if Rails.application.config.database %>`
and
`<% end %>`

instead of 

```
<%
   if Rails.application.config.database
%>
```

and

```
<%
  end
%>
```

Reply:
Absolutely. I changed the the conditional statements to single-line and renamed the tests.